components/AddQuestionButton/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styles from './AddQuestionButton.module.scss';
+
+interface AddQuestionButtonProps {
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+  href?: string;
+  className?: string;
+}
+
+const AddQuestionButton: React.FC<AddQuestionButtonProps> = ({
+                                                               onClick,
+                                                               href = '#',
+                                                               className = '',
+                                                             }) => {
+  const handleClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    if (onClick) {
+      onClick(event);
+    }
+  };
+
+  return (
+    <div className={`${styles.container} ${className}`.trim()}>
```

Comment:
L22: Looks like this <div> tag needs a `role=listitem` since it's inside a `role=list`. This is something that the free version of axe DevTools picked up.